content/_help/verify-your-identity/verify-your-identity-in-person._en.md

```diff
@@ -0,0 +1,112 @@
+
+
+
+If you’re unable to complete any of these steps, please contact the partner agency’s help center for support.   
+
+## Bring the following items to the Post Office
+
+### A copy of your emailed barcode
+
+Once you complete the steps on Login.gov, we’ll email you a barcode and a deadline to go to the Post Office. You can print the email, or show the email from your mobile device. If you go to the Post Office after the deadline, your information will not be saved and you will need to restart the process.  
+
+### Your state-issued ID
+
+Bring the same ID you used on Login.gov. Your ID must not be expired. At this time, only the following state-issued identification is accepted: 
+
+* Driver’s license from all 50 states, the District of Columbia (DC), and other US territories (Guam, US Virgin Islands, Mariana Islands and Puerto Rico)
+* A non-driver’s license state-issued ID card
+* * This is an identity document issued by the state, the District of Columbia (DC), or US territory that asserts identity but does not give driving privileges.
+
+We do not currently accept any other forms of identification, such as passports and military IDs. You cannot verify your identity at the Post Office without a state-issued ID. We’re currently working to add more ways to verify your identity.
+
+In the meantime, please contact the partner agency’s help center to find out what you can do.
+
+### Proof of your current address
+
+If your current address does not match the address on your ID, you will need to bring proof of your current address to the Post Office. Acceptable forms of proof of address are:
+
+* Lease, Mortgage, or Deed of Trust
+* Voter Registration
+* Vehicle Registration Card
+* Home or Vehicle Insurance Policy
+
+You can bring a physical copy or show a digital copy of the document. You cannot show a photo or screenshot of the document. 
+
+
+
+## Complete the process at the Post Office before your deadline
+
+### Available locations in the Washington D.C. area 
+
+You can visit any of the following Post Office locations. You do not need to visit the location you selected on Login.gov. 
+
+ 
+
+1. BALTIMORE — Post Office™\
+   900 E FAYETTE ST RM 118\
+   BALTIMORE, MD 21233-9715
+2. BETHESDA — Post Office™\
+   6900 WISCONSIN AVE STE 100\
+   CHEVY CHASE, MD 20815-9996
+3. FRIENDSHIP — Post Office™\
+   4005 WISCONSIN AVE NW\
+   WASHINGTON, DC 20016-9997
+4. WASHINGTON — Post Office™\
+   900 BRENTWOOD RD NE\
+   WASHINGTON, DC 20018-9997
+5. ARLINGTON — Post Office™\
+   3118 WASHINGTON BLVD\
+   ARLINGTON, VA 22201-9998
+6. ASHBURN — Post Office™\
+   44715 PRENTICE DR\
+   DULLES, VA 20101-9996
+7. MERRIFIELD — Post Office™\
+   8409 LEE HWY\
+   MERRIFIELD, VA 22116-9998
+
+### You can speak with any retail associate to verify your identity
+
+* ## Wait in line at the Post Office. The retail clerk will scan your barcode and review your documents.
```

Comment:
We need to remove the extra "##" so that this bullet note is not a heading

Reply:
> We need to remove the extra "##" so that this bullet note is not a heading

This should be fixed now in 705bdd9.